neat-form-core/src/main/java/com/nerdstone/neatformcore/views/builders/MultiChoiceCheckBoxViewBuilder.kt

```diff
@@ -11,15 +12,17 @@ import com.nerdstone.neatformcore.utils.Utils
 import com.nerdstone.neatformcore.utils.ViewUtils
 import com.nerdstone.neatformcore.utils.getViewsByTagValue
 import com.nerdstone.neatformcore.views.containers.MultiChoiceCheckBox
+import timber.log.Timber
 import java.util.*
 
 class MultiChoiceCheckBoxViewBuilder(override val nFormView: NFormView) : ViewBuilder {
 
     private val multiChoiceCheckBox = nFormView as MultiChoiceCheckBox
+    private var checkBoxTextSize: Float? = null
     private var valuesMap: HashMap<String, NFormViewData?>? = null
 
     enum class MultiChoiceCheckBoxProperties {
-        TEXT
+        TEXT, CHECK_BOX_OPTIONS_TEXT_SIZE,LABEL_TEXT_SIZE
```

Comment:
can we rename this CHECK_BOX_OPTIONS_TEXT_SIZE to this OPTIONS_TEXT_SIZE to reduce verbosity